java/build.gradle

```
@@ -102,7 +102,7 @@ targetCompatibility = 1.8
 
 group = "com.scalar-labs"
 archivesBaseName = "scalar-admin"
-version = "2.1.0"
+version = "2.2.0"
```

I understand that, by using `implementation`, this PR fixes the bug where the `complie` dependencies are not included in `pom.xml`.

So I think it's `bugfix`, and we can release it in the next patch release.

```suggestion
version = "2.1.1"
```

---

Thank you 🙇